Week2/homework/map_filter.js

```diff
@@ -2,4 +2,9 @@
 
 const numbers = [1, 2, 3, 4];
 
-// Add your code here
+var oddNumbers = numbers.filter(function(oddNumber){
+    return oddNumber%2==1
+})
+//console.log(oddNumbers);
+var doubledNumbers = oddNumbers.map((doubledNumber) => doubledNumber*2 )
+console.log('The doubled numbers are: ' + doubledNumbers);
```

Comment:
I like that you made each section separately, as it also helps with testing the intermediate results.  But you could also chain both operations together:
```
const doubleOdds = numbers.filter(n => n % 2 !== 0).map(x => x * 2);
```
And for fun, you could define this as a reduce, and only go through the sequence once. 😄 
I'll leave that as a challenge.  If you want help , just contact me on slack.

Reply:
Thanks for very useful tips Rohni, I'll do my best in revising me homework.